canvas_account_admin_tools/settings/local.py

```diff
@@ -57,6 +58,13 @@
          'type': 'All courses',
          'year': '2016',
       },
+      # TLT-2574 - Course Info People Add
+      'test_data_for_multiple_users_add': {
+         'canvas_role': 'Guest',
+         'unsuccessful_add': '12345678, 23456789',
```

Comment:
Nope, this one is meant to be a string (and is working as intended).  This is for the unsuccessful add, so we can just pass in a string of however many ids we want to pass in to the add user input field.  Unlike the other tests, there is no need to put it into a list and then join it, because there is no  clean up needed for this test (via rest api call, which removes it one at a time, where the list is needed).  So I want to pass this in as a string.  Confusing much, perhaps in the context of the other two data points?  I added comments already within the test itself, but I can add in a comment here if that's helpful...